pymongo/mongo_client.py

```diff
@@ -401,6 +401,10 @@ def __init__(
             "mongodb+srv://" URIs. Defaults to "mongodb". Use it like so::
 
                 MongoClient("mongodb+srv://example.com/?srvServiceName=customname")
+          - `srvMaxHosts`: (int) limits the number of mongos-like hosts a client will
+            connect to. More specifically, when a mongodb+srv:// connection string
```

Comment:
Nit: we put "mongodb+srv://" in quotes for `srvServiceName` above.

Reply:
Done.